eng/Version.Details.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="System.Device.Gpio" Version="0.1.0-prerelease.19352.2">
+    <Dependency Name="System.Device.Gpio" Version="0.1.0-prerelease.19353.3">
```

Comment:
I don't think you should be changing versions in this PR. cc: @joperezr

Reply:
Yup this is wrong. You shouldn’t change this in your PR. Looks like you may have includded other changes in your rebase.

Reply:
@joperezr interestingly the version is higher

Reply:
Well this is a puzzle for me. I synced my local master with a fetch and merge with your master. Then I rebased the change branch onto the end of the master branch. After that I pushed to my forked change.

The changes to the versions already appear in your master and so not sure why they are showing and history shows they came from another PR.

Anyway it might be quicker to just cancel this PR right now as I can create start again with little effort. What do you think ?

Reply:
@Frankenslag force pushing to your branch is equivalent to starting new PR